woocommerce/woocommerce-android
# Refactored payment info card section in Order Detail

This PR takes care of the fourth step of the Order Detail refactoring #2844 .

#### Changes
- Adds the Order Payment Info card (`OrderDetailPaymentInfoView`) to the Order detail screen.
- Adds 2 new variables to the `Order` model to find out if an order is paid or is awaiting payment.

#### Notes
- I have split this massive change into multiple small PRs in order to make reviews easier. 
- This PR is to be merged into a feature branch.
- This PR just adds the Order detail Payment info card to the Order detail screen. The remaining cards will be added in subsequent PRs along with unit tests and click actions for all the buttons in the Order detail.
- **This PR is in draft till this [Step 3 PR](https://github.com/woocommerce/woocommerce-android/pull/2860) can be merged.**

#### Testing
- Click on an order from the Orders tab.
- Verify that the Payment card is displayed correctly for an order. **Note that the `Refunds` section of the `Payment` card will take place in a subsequent PR since it requires information from the refunds API.**

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/93575114-dae49900-f9b6-11ea-91d0-2ce71e84a5d3.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/93575118-dc15c600-f9b6-11ea-9239-0db348656e0e.png" width="300"/>

Update release notes:
